src/component/Books.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+// import Book from './Book';
+import CreateBook from './CreateBook';
+
+const Books = () => (
+  <div>
+    {/* <Book />
+    <Book />
+    <Book /> */}
+    <CreateBook />
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] @Kwamay Great work implementing the changes requested, however, you ended up having a component that you are not using and commented some lines of code in it, kindly consider deleting this component as it's not in use.